meetings/2018-12-12.md

```diff
@@ -0,0 +1,110 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-12-05
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=gqSmf5szUfQ   
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/639
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Michaël Zasso @targos (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+
+## Agenda
+
+### Announcements
+
+* No annoucements this week
+
+### nodejs/node
+
+* cli: add --max-http-header-size flag [#24811](https://github.com/nodejs/node/pull/24811)
+  * Added to agenda for visibility.
+  * Plan is to land and backport sooner than later as people were affected by change in the
+    community release and this provides a work around.
+  * Removed from agenda for now.
+
+* Fixing child_process module to check values passed strictly to the options object [#24267](https://github.com/nodejs/node/pull/24267)
+  * controversial as there is disagreement in the issue as to whether this should land.
+  * Back to github and invite Liran to next TSC meeting to provide a deeper dive and then
+    we can decide on next steps.
```

Comment:
This is a disappointing result. This doesn't seem like a discussion/decision that needs to happen synchronously (and therefore necessarily exclude people).

Reply:
Meh, I suppose a case can be made to seed the broader policy discussion with a conversation first. Seems to suggest a lack of engagement to me, to be honest, but it's not unreasonable. I'm going to delete my comment.